README.md

```diff
@@ -15,17 +15,12 @@ This repository includes files with plain SQL that can be used to recreate a dat
 
```

Comment:
Please add a project title and features description in README as it looks more professional.